docs/devGuide/devGuide.md

```diff
@@ -202,6 +202,12 @@ You can add the `--fix` flag to correct any fixable style errors.
 $ ./node_modules/.bin/eslint . --fix
 ```
 
+To correct fixable style errors (JS and CSS), run
```

Comment:
You may also correct fixable style errors for both JavaScript and CSS, by running this script

Since the last 3 commands did not use npm scripts, let's make it more explicit here, that it is using npm scripts.